parlai/core/torch_agent.py

```diff
@@ -1364,6 +1361,11 @@ def batchify(self, obs_batch, sort=False):
         if any('image' in ex for ex in exs):
             imgs = [ex.get('image', None) for ex in exs]
 
+        kleenexs = [
```

Comment:
kleenexs as a variable name would probably be very confusing =/ Why are we stripping out the Tensors here?

Reply:
Yeah sadly I don’t think this should stay lol

Def add the comment on why we strip

Reply:
I'll add a comment, but long story short we must avoid passing tensors along multiprocessing queues as it results in an expensive tensor copy operation

the name is staying 😄